flash/image/classification/integrations/baal/dropout.py

```diff
@@ -15,10 +15,20 @@
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 
 import flash
-from flash.core.utilities.imports import _BAAL_AVAILABLE
+from flash.core.utilities.imports import _BAAL_AVAILABLE, _BAAL_GREATER_EQUAL_1_5_2
 
 if _BAAL_AVAILABLE:
-    from baal.bayesian.dropout import _patch_dropout_layers
+    # _patch_dropout_layers function was replaced with replace_layers_in_module helper
+    # function in v1.5.2 (https://github.com/ElementAI/baal/pull/194 for more details)
+    if _BAAL_GREATER_EQUAL_1_5_2:
```

Comment:
In the requirements we have pinned `baal>=1.5.2` so shouldn't we just let it fail if the baal version is older than that? Otherwise, if we can pinpoint exactly which version changed the bahaviour we could support older versions too (but that should be reflected in the requirements file)

Reply:
I see what you mean. We do support older versions of baal:

https://github.com/PyTorchLightning/lightning-flash/blob/788a791d6e4cdd1ba1d684855e86cc8ceaba4883/flash/image/classification/integrations/baal/dropout.py#L30-L31

It was changed in v1.5.2 as mentioned in the comment, and this can also go in the requirements fiile (comment). Do you think that will be enough? Or maybe, we can just mention `baal` instead of specifying a version, since we support all versions.

Reply:
Ah I see :smiley: Let's just set a more generous min version (1.3.2 maybe?)

Reply:
Yes, done! 🚀